fabric-textures-v0/src/main/java/net/fabricmc/fabric/mixin/client/texture/SpriteAccessor.java

```diff
@@ -0,0 +1,16 @@
+package net.fabricmc.fabric.mixin.client.texture;
+
+import net.minecraft.client.texture.NativeImage;
+import net.minecraft.client.texture.Sprite;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+/**
+ * Simply gets the {@link NativeImage[]} in Sprite.
+ * Don't shy away from casting to this if you need to.
+ */
```

Comment:
We usually don't reccomend casting to accessors since they are impl detail, if you want this exposed, it would be best to make a util method that calls this accessor in api.